src/lib.rs

```diff
@@ -558,3 +570,37 @@ pub fn triangulate(points: &[Point]) -> Triangulation {
 
     triangulation
 }
+
+#[inline]
+fn f64_abs(f: f64) -> f64 {
```

Comment:
Please consider defining this method twice and just using the standard cfg macro to pick which one gets compiled.
I believe that will be a bit cleaner because we can split each implementation based on whether nostd is used or not.

For example, inspired in https://github.com/georust/robust/blob/6b482661d48207ebb0ff1c4bdc74aad5c6fc0387/src/lib.rs#L1174
```rust
// f64::abs is not yet available in libcore, so we’ll need to use a separate
// crate for this functionality.
//
// https://github.com/rust-lang/rust/issues/50145
#[cfg(feature = "no_std")]
#[inline]
fn abs(x: f64) -> f64 {
   const SIGN_BIT: u64 = 1 << 63;
    f64::from_bits(f64::to_bits(f) & !SIGN_BIT)
}
#[cfg(not(feature = "no_std"))]
#[inline]
fn abs(x: f64) -> f64 {
    x.abs()
}
```